themes/radial/components/label.typ

```diff
@@ -11,7 +11,7 @@
   let data = entry-type-metadata.at(type)
   let colored-image = utils.change-icon-color(raw-icon: data.icon, fill: white)
 
-  box(fill: data.color, outset: 3pt, radius: 1.5pt)[
```

Comment:
Changing this will change every call to label as well, including the ones in the `title`  and `toc` components. Please expose the radius as a parameter, and then have it default to 1.5pt. Then you modify it when you need to call it for the cover.